index.html

```diff
@@ -1,9 +1,12 @@
 <!DOCTYPE html>
 <html>
-   <body>
-      <p>Radio check. Pilot, please confirm.</p>
-      <p>Confirmed. The geese done got us. What can I do ya for.</p>
-      <p>We've identified a choice river for you to "land" in, hope that's coooool with youuu? 🤪</p>
-      <p>Totally fine. No worries bro. Tell my wife I love her.</p>
-   </body>
+   <head>
+      <link rel="stylesheet" type="text/css" href="style.css">
+   </head>
+      <body>
```

Comment:
Uhmmm, _actually_ this isn't the correct indentation!! You REALLY need to fix this RIGHT NOW, okay??? 🧐

Reply:
Acceptable I SUPPOSE